src/langs/en/messages.json

```
@@ -145,5 +145,6 @@
   "NOT_A_GUEST": "Your account is not registered as a valid pseudonimized user",
   "ACCOUNT_NOT_FOUND": "An account was not found",
   "CANNOT_CREATE_MEMBERSHIP_CAUSE_ITEM_LOGIN_SCHEMA_EXISTS": "You cannot request a membership because the targeted ressource allows pseudonimized users",
-  "CANNOT_MODIFY_GUEST_ITEM_MEMBERSHIP": "Cannot modify the permission of targeted account because it is a pseudonimized user"
+  "CANNOT_MODIFY_GUEST_ITEM_MEMBERSHIP": "Cannot modify the permission of targeted account because it is a pseudonymized user",
+  "CANNOT_ENROLL_ITEM_WITHOUT_ITEM_LOGIN_SCHEMA": "Cannot enroll in the targeted item because it doesn't allow pseudonymized users"
```

What do you think about:

```suggestion
  "CANNOT_ENROLL_ITEM_AS_PSEUDONYMIZED": "Cannot enroll in the targeted item because it doesn't allow pseudonymized users"
```

---

It's okay to me to change, but I feel like the backend will have to deal with different name of the Item Login Schema / Guest Account.
I know that it will just move the issue in the frontend, but the API Schema doesn't mention any "pseudonymized" component, so for someone who try to read the Schema, he won't understand.

---

Mmmh okay, in this case just ignore my comment :)